cypress/integration/inventory/create-page-type-request.spec.js

```diff
@@ -83,7 +83,7 @@ describe('ui-inventory: Create page type request', () => {
     Requests.deleteRequestPolicyApi(requestPolicyId);
   });
 
-  it('C546: Create new request for "Page" type (folijet) (vega)', { tags: [TestTypes.smoke, DevTeams.folijet] }, () => {
+  it('C546: Create new request for "Page" type (folijet) (vega)', { tags: [TestTypes.smoke, DevTeams.folijet, DevTeams.vega] }, () => {
```

Comment:
why you mark this test for vega?
![image](https://user-images.githubusercontent.com/26041116/180935985-999e145a-4c44-43d5-b758-9909671fd03d.png)

Reply:
please don't add the tag for Vega it will be problem for me to find this test and to fix

Reply:
Initially it is the vega team test. Also the vega team is responsible for this functionality. So let's leave this tag for vega team and remove the tag from folijet team

Reply:
let's do as you suggest, remove tag for Folijet team